examples/material_gallery/lib/demo/markdown_demo.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_markdown/flutter_markdown.dart';
+
+class MarkdownDemo extends StatefulComponent {
+  MarkdownDemo({Key key}) : super(key: key);
+
+  _MarkdownDemoState createState() => new _MarkdownDemoState();
+}
+
+class _MarkdownDemoState extends State<MarkdownDemo> {
+
+  void initState() {
+    super.initState();
+
+    DefaultAssetBundle.of(context).loadString('packages/flutter_gallery_assets/markdown.md').then((String fileContents) {
+      setState(() {
```

Comment:
What happens if this object is disposed between the time we kick off the load and the time it completes?  You should add a check for `mounted` before calling `setState`.